src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -523,6 +527,67 @@ class SnykSettingsDialog(
                     fill = UIGridConstraints.FILL_VERTICAL,
                     hSizePolicy = 1,
                     vSizePolicy = UIGridConstraints.SIZEPOLICY_WANT_GROW,
+                    colSpan = 2,
+                    indent = 0,
+                ),
+            )
+
+            val netNewIssuesText =
+                JLabel(
+                    "<html><br/> Snyk: <b>All issues vs Net new issues</b> <br/>" +
+                        "Specifies whether to see only net new issues or all issues. " +
+                        "Only applies to Code Security and Code Quality.<br/><br/>" +
+                        "Note: this is an experimental feature. Please reach out to Snyk for more details:",
```

Comment:
I would not want to ask the user to reach out to Snyk support without reason. This should be removed, I think.

Reply:
Most likely, the design should be more concise, but let's wait for @neilm-snyk to weigh in.

Reply:
What did Neil say?

Reply:
Commenting here for visibility as well, the entire note line is more relevant to experimental features. As we're planning on moving this out of experimental I think we should remove the entire note line